modules/gathering-operator-logs.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+[id="gathering-operator-logs_{context}"]
+= Gathering Operator logs
+
+If you experience Operator issues, you can gather detailed diagnostic information from Operator Pod logs.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+* Your API service is still functional.
+* You have installed the OpenShift CLI (`oc`).
+* You have the fully-qualified domain names of the control plane, or master machines.
+
+.Procedure
+
+List the Operator Pods that are running in the Operator's namespace, plus the Pod status, restarts, and age:
++
```

Comment:
I think the `+` here is causing a formatting issue and can be removed.

Reply:
For some reason I can't see the second plus sign. I have added the missing dot to the start of the step and checked the formatting and it now looks ok.